### PR TITLE
Add Interval Check Procedure in Duplex Workflow

### PR DESCRIPTION
**Pull request by yueyaog**
This PR added @MicahR-Y's interval list checker to the Duplex workflows that we use in production. The updated WDLs have been tested in [PDO-30143_Crompton_20-535_ichor](https://app.terra.bio/#workspaces/broadtagteam/PDO-30143_Crompton_20-535_ichor/workflows) Terra workspace.

### CODE REVIEW

File: Liquid_Biopsy_Duplex_Analysis/GenerateDuplexConsensusBams/GenerateDuplexConsensusBams.wdl [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Import BaitSetNameCheck task
+import "https://raw.githubusercontent.com/broadinstitute/TAG-public/master/checkBaitSetName/checkBaitSetName.dev.wdl" as checkBaitSetName
```
Comment (fleharty) [inline Liquid_Biopsy_Duplex_Analysis/GenerateDuplexConsensusBams/GenerateDuplexConsensusBams.wdl:2]: I propose using: https://raw.githubusercontent.com/broadinstitute/TAG-public/v1.1.1/checkBaitSetName/checkBaitSetName.dev.wdl

Comment (fleharty) [reply Liquid_Biopsy_Duplex_Analysis/GenerateDuplexConsensusBams/GenerateDuplexConsensusBams.wdl:2]: Instead, can you use something like, import "../../../../../tasks/broad/JointGenotypingTasks.wdl" as Tasks

This way we are referring to the current version (but not the master).

File: Liquid_Biopsy_Duplex_Analysis/MakeCallsFromConsensus/MakeCallsFromConsensus.wdl [hunk 2]
```diff
@@ -1,5 +1,6 @@
 version 1.0
 import "https://raw.githubusercontent.com/broadinstitute/gatk/4.1.8.1/scripts/mutect2_wdl/mutect2.wdl" as m2
+import "https://raw.githubusercontent.com/broadinstitute/TAG-public/master/checkBaitSetName/checkBaitSetName.wdl" as checkBaitSetName
```
Comment (fleharty) [inline Liquid_Biopsy_Duplex_Analysis/MakeCallsFromConsensus/MakeCallsFromConsensus.wdl:3]: Let's not use master, use relative path.

**Review by fleharty — APPROVED** [review]
Minor changes requested, feel free to merge once those corrections are made.

**Comment by yueyaog** [comment]
PR comments have been addressed in the most recent commit. Currently testing the latest scripts in Terra. Will merge if they work.

**Pull request merged**